src/omero_cli_render.py

```diff
@@ -656,6 +668,7 @@ def set(self, args):
                 "Updated rendering settings for Image:%s" % img.id)
             if not args.skipthumbs:
                 self._generate_thumbs([img])
+            img._closeRE()
```

Comment:
In general, big :+1: for closing services, but:

 * if this needs to be here, probably best in a try/finally
 * I wonder though why your `gateway_require` decorators isn't handling this

Reply:
My understanding is that the decorator will  handle the closing of the gateway and its services at the end of the command. But I do not see anything here or in the gateway that would close each rendering enging within the loop.

As noticed by @dominikl and  myself, this becomes especially important when using `bin/omero render set` against a large collection of objects e.g. a plate or a screen.